entity-framework/core/what-is-new/ef-core-6.0/breaking-changes.md

```diff
@@ -944,3 +945,32 @@ public object Create(DbContext context, bool designTime)
         ? (context.GetType(), dynamicContext.UseIntProperty, designTime)
         : (object)context.GetType();
 ```
+
+The navigation '{navigation}' was ignored from 'Include' in the query since the fix-up will automatically populate it. If any further navigations are specified in 'Include' afterwards then they will be ignored. Walking back include tree is not allowed.
```

Comment:
```suggestion
The navigation '{navigation}' was ignored from 'Include' in the query since the fix-up will automatically populate it. If any further navigations are specified in 'Include' afterwards then they will be ignored. Walking back in the include tree is not allowed.
```